tests/driver_adxl345/main.c

```diff
@@ -31,14 +31,11 @@ int main(void)
     adxl345_t dev;
     adxl345_data_t data;
 
-    dev.i2c = ADXL345_PARAM_I2C;
-    dev.addr = ADXL345_PARAM_ADDR;
-
     puts("ADXL345 test application");
     printf("Initializing ADXL345 accelerometer at I2C_DEV(%i)... ",
-           dev.i2c);
+           adxl345_params->i2c);
```

Comment:
this should be `adx11345_params[0].i2c` if I am not mistaken. Comes down to the same thing, but would semantically the right thing to do here.